composer.json

```diff
@@ -19,11 +19,12 @@
         "php": "7.4.*",
         "ext-json": "*",
         "doctrine/annotations": "^1.10.0",
+        "symfony/config": "4.4.*",
         "symfony/dependency-injection": "4.4.*",
         "symfony/event-dispatcher": "4.4.*",
         "symfony/http-foundation": "4.4.*",
         "symfony/http-kernel": "4.4.*",
-        "symfony/config": "4.4.*"
+        "symfony/options-resolver": "^5.0"
```

Comment:
FYI, I think that we chose to require only Symfony `4.4` components, because, [starting with `5.2`, Symfony introduced their own "Rate Limiter" component](https://symfony.com/blog/new-in-symfony-5-2-rate-limiter-component)

so... we thought that this bundle would be helpful only for Symfony 4 projects 🤷 (not Symfony 3 because we didn't need to support such old versions)

but if you think that we will continue to use and maintain this RateLimit bundle, even after moving all our projects to Symfony 5... then I'd suggest to update all requirements to match `^4.4 || ^5.0` 🤷

Reply:
Humm . I don't know. It is true that symfony has its own ratelimiter, not sure it does what we do though. 
This dependency was because of @Brah0um  comment about verifying input 🤷‍♀️ we could chose to do this another way

Reply:
I'm agreed with Oliver, this bundle was build because Symfony Rate Limiter is only compatible with Symfony 5.2+

I think we should switch to `4.4.*|^5.0`